tests/forward_models/rft/test_gendata_rft.py

```diff
@@ -466,7 +466,7 @@ def test_defaults():
 
     fm_description_file = os.path.join(
         os.path.dirname(semeio.forward_models.__file__),
-        "config",
+        "config_deprecated",
```

Comment:
debugging stuff left in here.

Reply:
Removed the test, as it asserts a relationship between the previously used config files filenames and the names output by ERT